_scripts/instruction-widget/install.js

```diff
@@ -161,7 +161,7 @@ module.exports = function(context) {
     }
     if (context.distro == "opbsd"){
       context.portcommand = "letsencrypt/client";
-      context.package = "pkg_add letsencrypt";
+      context.package = "pkg_add certbot";
       context.base_command = "letsencrypt";
```

Comment:
Just to verify, is the command also called `certbot` now? If so, this line needs to be changed as well, however, I'm happy to do this myself in a follow up PR if you prefer.